tests/common.py

```diff
@@ -166,11 +166,15 @@ def __init__(self, domain=None, dependencies=[], setup=None,
 class MockPlatform(object):
     """Provide a fake platform."""
 
-    def __init__(self, setup_platform=None, dependencies=[]):
+    def __init__(self, setup_platform=None, dependencies=[],
```

Comment:
[Mutable default kwargs](http://reinout.vanrees.org/weblog/2012/04/18/default-parameters.html) are generally speaking a really bad idea ie:

```python
(ve) [jeff@omniscience home-assistant] (test-better-sonos)$ python3
Python 3.4.3 (default, Jun 29 2015, 12:16:01) 
[GCC 5.1.1 20150618 (Red Hat 5.1.1-4)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> def append_one(data=[]):
...     data.append(1)
...     return data
... 
>>> append_one()
[1]
>>> append_one()
[1, 1]
>>> append_one()
[1, 1, 1]
>>> append_one()
[1, 1, 1, 1]
>>> append_one(data=[2])
[2, 1]
>>> append_one(data=[2])
[2, 1]
>>> append_one(data=[2])
[2, 1]
>>> append_one()
[1, 1, 1, 1, 1]
>>> append_one()
[1, 1, 1, 1, 1, 1]
```

May I suggest changing `dependencies=[]` to `dependencies=None` or something else?